models/demos/metal_BERT_large_11/tt/bert_encoder.py

```diff
@@ -142,13 +143,13 @@ def __init__(self, config, encoder_idx, state_dict, device, model_config, tt_cac
         if "OP7_SELFOUT_CONFIG" in model_config:
 
             def op7_mm_plus_bias(mha_res, attention_output_weight, attention_output_bias):
-                mha_out = tt_lib.operations.primary.matmul(
+                mha_out = ttnn.linear(
```

Comment:
can you educate me on the difference between `matmul` and `linear`?

Reply:
I see that in some cases we change to `linear` and sometimes on `matmul`

Reply:
They're the same op under the hood. In terms of the ttnn calls, the main differences are that linear 
- has two extra parameters: `bias` and `activation`
- does not propagate batching of b to the op (while matmul does)

Reply:
Thank you!